libs/vm-modules/examples/fixedpoint/fixed_point.etch

```diff
@@ -39,5 +59,39 @@ function main()
   printLn("exp(" + toString(d) + ") = " + toString(exp(d)));
   printLn("sin(" + toString(pi) + ") = " + toString(sin(pi)));
 
+  var pi128 : Fixed128 = 3.1415fp128;
+  printLn("pi = " + toString(pi128));
+
+  var x: Fixed128 = -pi128;
+  var y = abs(x);
+  printLn("x = " + toString(x));
+  printLn("y = " + toString(y));
+  printLn("abs(x) = " + toString(y));
+  printLn("exp(x) = " + toString(exp(x)));
+  printLn("exp(abs(x)) = " + toString(exp(y)));
+  printLn("log2(y) = " + toString(log2(y)));
+  printLn("log(y) = " + toString(log(y)));
+  printLn("log10(y) = " + toString(log10(y)));
+  printLn("sqrt(y) = " + toString(sqrt(y)));
+  printLn("sin(y) = " + toString(sin(y)));
+  printLn("cos(y) = " + toString(cos(y)));
+  printLn("tan(y) = " + toString(tan(y)));
+  x = sin(y);
+  printLn("x = " + toString(x));
+  printLn("asin(x) = " + toString(asin(x)));
+  x = cos(y);
+  printLn("x = " + toString(x));
+  printLn("acos(x) = " + toString(acos(x)));
+  x = tan(y);
+  printLn("x = " + toString(x));
+  printLn("atan(x) = " + toString(atan(x)));
+
   printLn("pow(" + toString(d) + ", " + toString(d) + ") = " + toString(pow(d, d)));
+
+  var data_shape = Array<UInt64>(2);
+  data_shape[0] = 2u64;
+  data_shape[1] = 4u64;
+  var T = Tensor(data_shape);
```

Comment:
Sorry to nitpick here, but lowercase for variables please.

Reply:
fixed